platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/chassis.py

```diff
@@ -289,6 +301,9 @@ def get_reboot_cause(self):
         """
         return self.REBOOT_CAUSE_NON_HARDWARE, ''
 
+    def get_all_components(self):
```

Comment:
implemented in base class
https://github.com/Azure/sonic-platform-common/blob/5da31e190d0df1ba58078cf182f6a3b3918253ab/sonic_platform_base/chassis_base.py#L169